package.json

```
@@ -92,7 +93,11 @@
     "debug": "^3.2.6",
     "eventbusjs": "^0.2.0",
     "js-cookie": "^2.2.1",
+    "msw": "^0.36.0",
```

Should be a dev dependency

```suggestion
    "msw": "^0.36.0",
```